### ISSUE TITLE
Make label.png label colors same for all labels.png for same image folder

### ISSUE THREAD

**Issue by dpetrini** [wkentaro/labelme#721]
First: this is a great job and very useful. Below is an improvement suggestion.

When we start program with options:
labelme data_annotated/ --labels labels.txt 

I inform a folder with images to be labeled and a file with previous labels names.

If I run 
labelme_json_to_dataset image1.json -o image1

I create a folder with the image, label mask and others files.

Thing is, if I use all labels from my label.txt file to label a image, is ok, it generates label.png file with all labels, ok.
On the other hand, in a second image, if I don't use all labels, when it generates the label.png image, there will be different colors for labels in both images.

This happens because the amount of labels from initial set is different in each image.

This is bad for semantic segmentation labeling as often some images doesn't have all labels and the resulting label.png images will be all different.

Suggestion:
Use the original set of labels in txt file "labels.txt" to create unique colors to be used when generating label.png's image. Label colors should be by folder, not by image.

This is great tool and I think will be very popular with these small details fixed.

Thank you very much.

ps. I made an ugly workaround inserting a very small label with the missing class in the image. Then when run labelme_json_to_dataset it disconsider the small label for image creating (I guess because is very small) but counts the label for color selection.

**Comment by wkentaro** [comment]
See advanced examples:
- https://github.com/wkentaro/labelme/tree/master/examples/semantic_segmentation
- https://github.com/wkentaro/labelme/tree/master/examples/instance_segmentation

**Comment by dpetrini** [comment]
Hi,

I tried to test directly with labelme2voc.py but as I first run labelme with different parameters it didn't work. But I saw the folders it generates in you example here in github, and I guess it respects one color for each label as I could see in examples.
So I think is solved and I will use like that in next project.

Thank you for you answer.
Best regards and thank you for your time writing this tool.

Daniel
